src/main/java/org/dcsa/bkg/model/transferobjects/BookingSummaryTo.java

```diff
@@ -0,0 +1,72 @@
+package org.dcsa.bkg.model.transferobjects;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import org.dcsa.core.events.model.enums.*;
+
+import java.time.OffsetDateTime;
+import java.util.UUID;
+
+@Data
+public class BookingSummaryTo {
```

Comment:
Personally I would like Transfer Object to be shortened with T**O** and not T**o**